kotlin-react/src/main/kotlin/react/RBuilder.kt

```diff
@@ -140,6 +140,7 @@ open class RBuilderImpl : RBuilder {
 
 open class RBuilderMultiple : RBuilderImpl()
 
+@Deprecated(message = "Unsafe API, use createElement")
```

Comment:
How am I supposed to use it if `createElement` doesn't have the `builder` parameter?